README.md

```diff
@@ -77,6 +77,8 @@ You can also choose to run it in background (detached mode):
 $ docker-compose up -d
 ```
 
+*Please note: You may need to run `docker-compose build` first*
```

Comment:
Makes sense. I would just prefer this:

- put the note right underneath the **Bringing up the stack** title
- append _"in case you switched from on branch to another or updated any base image, ..."_
- respect the **NOTE:** format used in other places of the README (eg. line 109)